stories/Atom/Navigation/Breadcrumb/Breadcrumb.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import './breadcrumb.scss';
 
-export const Breadcrumb = ({ style, text, ...props }) => {
+export const Breadcrumb = ({ text }) => {
   return (
-    <span className={['breadcrumb_item',`breadcrumb_item--${style}`].join(' ')}   {...props}><a href="#">{text}</a></span>
+    <span className={'breadcrumb__item'}><a href="#">{text}</a></span>
```

Comment:
@harishkumar-srijan can you please remove span and we can keep only <a> tag

Reply:
Done